### DISCUSSION TITLE
how to leave contributing to education/series-intro-to-github-flow?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#121410] [New to GitHub]
last year I had joined to github course for beginners. few months later I found that I couldn't get out of contributing. by the way there is no information how to do it by myself. please, help t_t
<img width="1240" alt="Screenshot 2024-04-28 at 17 08 43" src="https://github.com/community/community/assets/68105353/ee183e4e-c4b9-4c58-8057-d68b3fd0eb26">

**Comment by gurjeetsinghvirdee** [comment]
hey @dbkarashev 
You are already completed the course, if you want to remove this from your repositories you can delete it by going through `Settings` Tab, then scroll down you will get an option to delete this repository.

**Reply by ghost** [reply]
how can i do it? idk where is course rep
<img width="1440" alt="Screenshot 2024-05-01 at 14 48 28" src="https://github.com/community/community/assets/68105353/47766a55-cc72-44bc-9a87-d7cdaf39ed1c">

**Reply by gurjeetsinghvirdee** [reply]
Okay, I understand now. That's not your repository; it was a project created by GitHub Classroom as part of an assignment. You can find it at https://github.com/Exp-Intro-to-GitHub-Flow-Cohort-1/series-intro-to-github-flow-dbkarashev. Since you've completed the course, you can just leave it as it is. It won't show up in your repository because it was not part of your project; rather, it was created as an assignment within the classroom's repository.

**Reply by pedrobritx** [reply]
Ok, but how do I delete it?

**Comment by marinsucks** [comment]
Hey, does anyone have updates on this ? the repo still appears in my repositories (pic 1). I think that like op, I was part of the organisation `Exp-Intro-to-GitHub-Flow-Cohort-1`, and since I left it I don't have access to the repo settings to remove myself anymore (pic 2). I have tried to join the organization again but I'm not allowed.

<img width="322" alt="Screenshot 2025-01-07 at 12 48 17" src="https://github.com/user-attachments/assets/1712189f-f59f-4175-9dd2-9beea1830eab" />
<img width="1435" alt="Screenshot 2025-01-07 at 12 48 36" src="https://github.com/user-attachments/assets/72ed5c5d-7d77-4c59-9875-d8e263e496a1" />
